src/templates/CakeConfigTemplate.js

```diff
@@ -54,50 +54,27 @@ const CakeConfigTemplate = ({ cards }) => {
       />
       <Heading>VYSKLADAJTE SI SVOJU TORTU NA MIERU</Heading>
       <ContentWrapper>
-        <Paragraph margin="0 35%" mobileMargin="0 5%" tabletMargin="0 15%" textAlign="center">
-          Na základe našej ponuky nižšie nám prosím písomne zadajte Vašu objednávku cez email, sms
-          alebo na naše sociálne siete.
-        </Paragraph>
-        <ContactWrapper>
-          <Paragraph fontFamily={theme.fontFamilies.dinPro.bold} lineHeight="24px">
-            <SocialButton marginRight="12px" icon="mail" />
+        <Paragraph margin="0 30%" mobileMargin="0 5%" tabletMargin="0 15%" textAlign="center">
+          Vyskladajte si Vašu tortu a objednávku prosím zadajte e-mailom na adresu
+          <Paragraph display="contents" color="blue">
+            {' '}
             makronkovo@makronkovo.sk
           </Paragraph>
-          <Paragraph fontFamily={theme.fontFamilies.dinPro.bold} lineHeight="24px">
-            <SocialButton marginRight="12px" icon="telephone" />
-            0948 009 800
-          </Paragraph>
+          . Objednávka je považovaná za záväznú našim potvrdením.
```

Comment:
tato bodka je tam tiez divne

Reply:
Je to tak spravené kvôli tomu, aby tá bodka nebola modrá ako text predtým. Lebo zvýraznený má byť len mail